### PR TITLE
Upcoming delete icon

### PR DESCRIPTION
**Pull request by brettjforsyth**
hotfix for the delete icon #595

### CODE REVIEW

File: plugins/events-manager/buddypress/profile.php [hunk 1]
```diff
@@ -250,7 +250,7 @@ class="events-table">
 								if ( ! empty( $attributes['Registration Link'] ) ) {
 									$maybe_url = eypd_maybe_url( $attributes['Registration Link'] );
 								};
-								$link = ( $maybe_url ) ? "<a href='{$maybe_url}' target='_blank'>Contact Organizer <i class='glyphicon glyphicon-new-window'></i></a>" : "<a href='{$EM_Event->guid}'>Contact Organizer</a>";
+								$link = ( $maybe_url ) ? "<a href='{$maybe_url}' target='_blank'>Contact Organizer <i class='fa fa-new-window'></i></a>" : "<a href='{$EM_Event->guid}'>Contact Organizer</a>";
```
[the remark this answers is not in the document]
Comment (brettjforsyth) [reply plugins/events-manager/buddypress/profile.php]: Gah. I was trying the alt before and that didn’t work and this did will fix later

[the remark this answers is not in the document]
Comment (brettjforsyth) [reply plugins/events-manager/buddypress/profile.php]: Is there a reason why we are using FA 4.7 and not 5?

File: plugins/events-manager/buddypress/profile.php [hunk 2]
```diff
@@ -267,7 +267,7 @@ class="events-table">
 											'booking_id' => $EM_Booking->booking_id,
 											'_wpnonce'   => $nonce,
 										] );
-										$cancel_link = '<a class="em-bookings-cancel" href="' . $cancel_url . '" onclick="if( !confirm(EM.booking_warning_cancel) ){ return false; }"><i class="glyphicon glyphicon-trash"></i></a>';
+										$cancel_link = '<a class="em-bookings-cancel" href="' . $cancel_url . '" onclick="if( !confirm(EM.booking_warning_cancel) ){ return false; }"><i class="fa fa-trash"></i></a>';
```
[the remark this answers is not in the document]
Comment (brettjforsyth) [reply plugins/events-manager/buddypress/profile.php]: Weird this doesn't work for you. I am hasitant to use the -o as it won't work in 5 and will have to include a 4 shim file to keep it working. 
https://fontawesome.com/v4.7.0/icon/trash should work according to the 4.7 docs. And its working on my machine. Could there be a version issue on your end?
screen shot 2018-11-29 at 9 59 25 am

@mandily-p This icon is left aligned like everything else in this table but do you want to center it to give it a bit of breathing room from the contact organizer icon (which doesn't always show as an aside).

[the remark this answers is not in the document]
Comment (brettjforsyth) [reply plugins/events-manager/buddypress/profile.php]: changed it to -o

[the remark this answers is not in the document]
Comment (bdolor) [reply plugins/events-manager/buddypress/profile.php]: you are correct, fa-trash works. Reverting to fa-trash and single quotes. will merge changes on local and push to dev.

**Pull request merged**